test/nuts/deployDestructive.nut.ts

```diff
@@ -20,6 +20,7 @@ describe('source:deploy --destructive NUTs', () => {
     const pathToClass = path.join(testkit.projectDir, output, `${apexName}.cls`);
     execCmd(`force:apex:class:create --classname ${apexName} --outputdir ${output} --api-version 58.0`, {
       ensureExitCode: 0,
+      cli: 'sf',
```

Comment:
if we're using `cli: 'sf'` we can remove PDR as a dev-dep and oclif dev-plugin right?